po/messages.pot

```diff
@@ -2113,6 +2322,257 @@ msgctxt "dlgSgAbout.tle"
 msgid "About SmartGit"
 msgstr ""
 
+msgctxt "dlgSgAiAnnotatorDropAllNotes.btn:"
+msgid "Drop"
+msgstr ""
+
+msgctxt "dlgSgAiAnnotatorDropAllNotes.fur"
+msgid "This will remove all AI-related Git notes from the repository."
+msgstr ""
+
+msgctxt "dlgSgAiAnnotatorDropAllNotes.hdl"
+msgid "Do you want to drop all AI-related Git notes?"
+msgstr ""
+
+msgctxt "dlgSgAiAnnotatorDropAllNotes.tle"
+msgid "Drop AI Notes"
+msgstr ""
+
+msgctxt "dlgSgAiApiKeyRepoLlmConsent.btn:"
+msgid "Allow"
+msgstr ""
+
+msgctxt "dlgSgAiApiKeyRepoLlmConsent.btn:"
+msgid "Deny"
+msgstr ""
+
+#| msgid "This is the first time that SmartGit will access the AI service for this repository:\\n\\n/Users/tomohiro/Git-projects/smartgit-translations-25.1-wip\\n\\nPlease confirm whether uploads to OpenAI are allowed. If allowed, SmartGit may upload repository data required by the AI feature you invoke \\(for example: diffs, commit messages, and related metadata\\).\\n\\nThis is a one-time safety check, and your choice will apply to all future requests for this repository with this AI service."
+msgctxt "dlgSgAiApiKeyRepoLlmConsent.fur"
+msgid "This is the first time that SmartGit will access the AI service for this repository:\\n\\n$1\\n\\nPlease confirm whether uploads to OpenAI are allowed. If allowed, SmartGit may upload repository data required by the AI feature you invoke \\(for example: diffs, commit messages, and related metadata\\).\\n\\nThis is a one-time safety check, and your choice will apply to all future requests for this repository with this AI service."
```

Comment:
This is actually:

"This is the first time that SmartGit will access the AI service for this repository:\n\n" + rootFile + "\n\nPlease confirm whether uploads to " + typeName + " are allowed. " + details + "\n\nThis is a one-time safety check, and your choice will apply to all future requests for this repository with this AI service."